examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>1) <s>However,</s> You</p><p> </p><p>2) that we <s>have </s>downloaded in <s>one of </s>the first cell of this ..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>